R/MltplxExperiment.R

```diff
@@ -32,10 +32,11 @@
 #' @import tidyr
 new_MltplxExperiment = function(x, y, marks, slide_id, window_sizes = NULL,
                                 ps = NULL, bw = NULL,
-                                dist_metric = NULL, metadata = NULL){
+                                dist_metric = NULL, metadata = NULL,
+                                windows = NULL){
```

Comment:
Can we add windows parameter to documentation and update?

Reply:
Done